scripts/obtain/dotnet-install.ps1

```diff
@@ -47,7 +47,7 @@
 #>
 [cmdletbinding()]
 param(
-   [string]$Channel="beta",
```

Comment:
@piotrpMSFT @krwq - if `dotnet-install` is the "real one" going forward, I'm gonna merge this, and then make a new PR to remove the old one.

Everyone cool with that?